### DISCUSSION TITLE
Need guidance in addition of a button in Prawn pdf

### DISCUSSION THREAD

**Discussion by PratibhaYadav01** [prawnpdf/prawn#1348] [Q&A]
can a Prawn pdf have a button which when on click it will navigate to a different site or a page in a project(a login page or home page maybe)?
I tried adding a button but it is not supporting and the button is not clickable.
Does Prawn library supports navigation from the pdf to another site or a page in the project itself?

**Comment by pointlessone** [comment]
Prawn provides links you can use those to direct readers elsewhere, either within the document or without. Conceptually it's just a region on the page so you can draw whatever you want in that region. Making it react to clicks visually is probably technically within PDF spec but Prawn doesn't support that functionality.
